code/modules/research/nanites/extra_settings/text.dm

```diff
@@ -7,6 +7,9 @@
 /datum/nanite_extra_setting/text/set_value(value)
 	src.value = trim(value)
 
+/datum/nanite_extra_setting/text/get_value()
```

Comment:
Don't you mean get_copy here and below. I'm not sure how this code works, but get_value returning a copy of itself doesn't seem correct ?

Reply:
good catch, yeah that's what I meant to put here. get_value isn't really used much as a micro optimization I probably shouldn't have done but it behaving as intended is definitely what's wanted